Mage.Client/src/main/resources/card-pictures-tok.txt

```diff
@@ -1357,4 +1358,13 @@
 |Generate|TOK:THB|Wolf|||WolfToken|
 |Generate|TOK:THB|Nightmare|||AshiokNightmareMuseToken|
 |Generate|TOK:THB|GoldToken|||Gold|
-|Generate|TOK:THB|ArtifactWallToken|||Wall|
\ No newline at end of file
+|Generate|TOK:THB|ArtifactWallToken|||Wall|
+|Generate|TOK:IKO|Beast|||BeastToken|
+|Generate|TOK:IKO|Cat Bird|||CatBirdToken|
+|Generate|TOK:IKO|Cat|||CatToken|
+|Generate|TOK:IKO|Dinosaur Beast|||DinosaurBeastToken|
+|Generate|TOK:IKO|Dinosaur|||DinosaurHasteToken|
+|Generate|TOK:IKO|Feather|||FeatherToken|
+|Generate|TOK:IKO|Human Soldier|||HumanSoldierToken|
```

Comment:
There are 3 different human soldiers, not one. Must be:
* `|Human Soldier|1|`
* `|Human Soldier|2|`
* `|Human Soldier|3|`

Also `HumanSoldierToken` class is not completed -- random token image index missing for IKO set. Look at `FoodToken` -- must be same with some modifications:
``` java
@Override
    public void setExpansionSetCodeForImage(String code) {
        super.setExpansionSetCodeForImage(code);

        if (getOriginalExpansionSetCode() != null && getOriginalExpansionSetCode().equals("ELD")) {
            setTokenType(RandomUtil.nextInt(4) + 1); // 1...4
        }
    }
```